jdaviz/configs/imviz/tests/test_wcs_utils.py

```diff
@@ -132,7 +136,9 @@ def test_non_wcs_layer_labels(self):
         assert self.imviz.app.data_collection[3].label == 'ndd'
 
         # Confirm that all data in collection are labeled.
-        assert len(self.imviz.app.state.layer_icons) == 4  # 3 + 1
+        assert len(self.imviz.app.data_collection) == 4  # 3 + 1
+        # NOTE: should be 4 once wcs-only layer is parsed through loaders
+        assert len(self.imviz.app.state.layer_icons) == 3  # 4

```

Comment:
NOTE: I did create a follow-up ticket for this.  WCS-only layers created through the orientation plugin do work as expected, but loading them manually through `load_data` will not result in an assigned icon.  See [JDAT-5548](https://jira.stsci.edu/browse/JDAT-5548)